sanic/response.py

```diff
@@ -317,13 +333,37 @@ async def file(
     :param mime_type: Specific mime_type.
     :param headers: Custom Headers.
     :param filename: Override filename.
+    :param last_modified: The last moditied date and time of the file.
+    :param max_age: Max age for cache control.
     :param _range:
     """
     headers = headers or {}
     if filename:
         headers.setdefault(
             "Content-Disposition", f'attachment; filename="{filename}"'
         )
+    if isinstance(last_modified, datetime):
+        last_modified = last_modified.timestamp()
+
+    if last_modified is None:
+        last_modified = Path(location).stat().st_mtime
+    if isinstance(max_age, Default):
+        max_age = 0  # Should change this (default) value to configable?
+
+    if last_modified:
+        headers.setdefault(
+            "last-modified", formatdate(last_modified, usegmt=True)
+        )
+    if max_age:
+        headers.setdefault("cache-control", f"max-age={max_age}")
+        headers.setdefault(
+            "expires",
+            formatdate(
+                time() + max_age,
+                usegmt=True,
+            ),
+        )
```

Comment:
Stylistic thing: can we keep the spacing between all of these ifs consistent?

Reply:
I am sure what are the spacing... Can you tell me a little more about it? What spacings should be consistent? Current spacing is made by Black via `make pretty`.

Reply:
Really? 
![image](https://user-images.githubusercontent.com/166269/169959863-c4df4bc6-1f41-430b-9f4a-1706501d9713.png)

If black is happy with it, then I will not question it. :shrug:

Reply:
GitHub Linter Checks workflow seems passed, and my local `tox -e lint` is an okay... 
Maybe there are some changes in the latest black version?